types.go

```diff
@@ -97,7 +97,7 @@ var FileTypeFields = map[FileType][]string{
 		"comment",
 	},
 
-	ModelsFile: []string{
+	ModelsFile: {
```

Comment:
Although it is unclear, these fields are considered required for the file detection. I don't consider these fields to be a hard requirement.